SPECS-EXTENDED/python-uamqp/python-uamqp.spec

```diff
@@ -24,7 +24,7 @@ BuildRequires:  %{py3_dist setuptools}
 %if %{with_check}
 BuildRequires:  %{py3_dist certifi}
 BuildRequires:  %{py3_dist pytest}
-BuildRequires:  %(py3_dist pytest-asyncio)
+BuildRequires:  %{py3_dist pytest-asyncio}
```

Comment:
I'd still increment the `Release` tag and add a changelog entry to be safe - we already published some of our SRPMs and this is a spec change, which should trigger a separate SRPM.

Reply:
Yes, I think it makes sense to do the ceremony :)